futures/README.md

```diff
@@ -43,7 +43,7 @@ fn main() {
       // connect() returns a future of an AMQP Client
       // that resolves once the handshake is done
       lapin::client::Client::connect(stream, &ConnectionOptions::default())
-    }).and_then(|client| {
+    }).and_then(|(client, )| {
```

Comment:
I made that a tuple so that you have to either deal with it or explicitely ignore it, instead of it being hidden and easily forgettable